frontend/Gemfile

```diff
@@ -1,7 +1,7 @@
 eval(File.read(File.dirname(__FILE__) + '/../common_spree_dependencies.rb'))
 
-gem 'solidus_core', path: '../core'
-gem 'solidus_api', path: '../api'
 gem 'generator_spec', group: :test
+gem 'solidus_api', path: '../api'
+gem 'solidus_core', path: '../core'
```

Comment:
@Empact @mamhoff good points. I've reverted the Gemfile change. And I'll make a followup PR with the removal of the cop. 👍